autocomplete/bash_autocomplete

```diff
@@ -3,14 +3,19 @@
 : ${PROG:=$(basename ${BASH_SOURCE})}
 
 _cli_bash_autocomplete() {
+  if [[ "${COMP_WORDS[@]:0:$COMP_CWORD}" != "source" ]]; then
     local cur opts base
     COMPREPLY=()
     cur="${COMP_WORDS[COMP_CWORD]}"
-    opts=$( ${COMP_WORDS[@]:0:$COMP_CWORD} --generate-bash-completion )
+    if [[ $cur == -* ]]; then
+      opts=$( ${COMP_WORDS[@]:0:$COMP_CWORD} ${cur} --generate-bash-completion )
+    else
+      opts=$( ${COMP_WORDS[@]:0:$COMP_CWORD} --generate-bash-completion )
```

Comment:
The same for this one :point_up:

Reply:
this seems false positive. `${COMP_WORDS[@]:0:$COMP_CWORD}` in not part for any `[[ ]]` and it is just using shell parameter expansion to get rid of what ever characters there are after current position of cursor to. This is already part current code and It completes current word/flag the cursor at instead of last word/flag in cli

Reply:
Ah I think shellcheck was complaining about double quoting to avoid word splitting. :)

Reply:
that wont be an issue here I guess due to `$( ... )`

Reply:
You still want to quote it I think, as if the word contains quotes this will blow up?

Reply:
I don't think so that's the purpose of that command to invoke it